### PR TITLE
RC2 release used version 4.6.*. However RTM release started to use th…

### PR DESCRIPTION
**Pull request by rahku**
…e default values of 1.0.* by mistake. Revert back to using 4.6 as major & minor version number for binaries

### CODE REVIEW

**Comment by rahku** [comment]
Fixes #6559

**Comment by rahku** [comment]
PTAL @gkhanna79 
Verified that windows binaries now get 4.6.buildNum.0 version

File: dir.props [hunk 1]
```diff
@@ -8,6 +8,12 @@
     <OsEnvironment Condition="'$(OsEnvironment)'==''">$(OS)</OsEnvironment>
   </PropertyGroup>
 
+  <!-- Version numbers for both managed & native binaries -->
+  <PropertyGroup>
```
Comment (gkhanna79) [inline dir.props]: To confirm, all binaries in CoreCLR bin folder are now having file version as 4.6, right?

CC @weshaggard

Comment (rahku) [inline dir.props]: yes all that are built ( which is all except coredistools.dll)

Comment (gkhanna79) [inline dir.props]: What is "coredistools.dll" and who is publishing it to our bindir?

Comment (rahku) [inline dir.props]: hmmm testbuild seems to be polluting the product binaries folder. https://github.com/dotnet/coreclr/blob/master/build-test.cmd#L107
This was added by @sejongoh

Comment (rahku) [inline dir.props]: This is disassembler tool probably used by gcstress and maybe other scenarios.

Comment (gkhanna79) [inline dir.props]: Can you please look into fixing this "pollution" ?

Comment (rahku) [inline dir.props]: can I commit this and open another issue for it. I believe this does not need to be blocked.

Comment (gkhanna79) [inline dir.props]: Sounds good!

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**